### DISCUSSION TITLE
how-to-stay-current-in-web-technology/

### DISCUSSION THREAD

**Discussion by giscus** [natelandau/natelandau.com#82] [Comments]
# how-to-stay-current-in-web-technology/

I am not a writer. I don't have this blog to write. I have it to teach me how to code and keep current in technology. You should too.

https://natelandau.com/how-to-stay-current-in-web-technology/

**Comment by stefancaliaro** [comment]
100% recommend this read. Despite having 7 years experience with WordPress and some static sites, I've struggled to pick a platform for self-hosting as well as whether to blog by my name or using some other alias/own company name. I've done a bit of both on LinkedIn https://www.linkedin.com/today/author/stefancaliaro?trk=article-ssr-frontend-pulse_more-articles + at https://siteroo.com.au/blog/

**Comment by stefancaliaro** [comment]
I like the WordPress like button, but don't like WordPress reliability at least out of the box.
